settings/profcond.setting.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ *
+ * @package CRM
+ * @copyright CiviCRM LLC (c) 2004-2013
+ *
+ */
+/*
+ * Settings metadata file
+ */
+return [
+  'menubar_position' => [
```

Comment:
What does 'menubar_position' mean in this context?

Reply:
Oops - that was a copy/paste error.  Fixed.